boards/microchip/pic32cxsg41_cult/doc/index.rst

```diff
@@ -0,0 +1,231 @@
+.. zephyr:board:: pic32cxsg41_cult
+
+Overview
+********
+
+The PIC32CX SG41 Curiosity Ultra evaluation kit is ideal for evaluation and
+prototyping with the PIC32CX SG Cortex®-M4F processor-based
+microcontrollers. The kit includes Microchip’s Embedded Debugger (EDBG),
+which provides a full debug interface without the need for additional
+hardware.
+
+Hardware
+********
+
+- PIC32CX1025SG41 ARM Cortex-M4F processor at 120 MHz
+- 32.768 kHz crystal oscillator
+- 12 MHz crystal oscillator
+- 1024 KiB flash memory and 256 KiB of RAM
+- Two yellow user LEDs
+- Two mechanical user push buttons
+- One reset button
+- On-board USB based EDBG unit with serial console
+- Embedded Debugger MCU (PKoB4)
+- 64 Mbit QSPI Flash
+- AT24MAC402 serial EEPROM with EUI-48™ MAC address
+- Ethernet transceiver 10/100 Mbps Ethernet MAC,
+  compatible with the IEEE 802.3 standard.
+- Arduino Uno header connectors
+- X32 Audio Interface Headers
+- mikroBUS header connectors
+- DAC Output header
+- USB interface, host, and device
+- SD/SDIO card connector
+
+Supported Features
+==================
+
+The ``pic32cxsg41_cult`` board supports the following hardware
+
++---------------+------------+----------------------------+
+| Interface     | Controller | Driver/Component           |
++===============+============+============================+
+| ADC           | on-chip    | adc                        |
++---------------+------------+----------------------------+
+| DAC           | on-chip    | dac                        |
++---------------+------------+----------------------------+
+| DMAC          | on-chip    | dma                        |
++---------------+------------+----------------------------+
+| EEPROM        | i2c        | eeprom, EUI-48 MAC Address |
++---------------+------------+----------------------------+
+| EIC           | on-chip    | interrupt_controller       |
++---------------+------------+----------------------------+
+| GMAC          | on-chip    | ethernet, mdio             |
++---------------+------------+----------------------------+
+| GPIO          | on-chip    | gpio                       |
++---------------+------------+----------------------------+
+| MPU           | on-chip    | arch/arm                   |
++---------------+------------+----------------------------+
+| NVIC          | on-chip    | arch/arm                   |
++---------------+------------+----------------------------+
```

Comment:
replace whole thing with the hw directive

Reply:
Still not addressed 
This needs to use `zephyr:board-supported-hw::` auto generated table. Check board template or other boards if you're not sure